myModule.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
yes, please delete the file -- any files needed for testing should be 1) created in a tmpdir and 2) deleted after the test (`on.exit(unlink(file))`)